### DISCUSSION TITLE
Case I want to handle.

### DISCUSSION THREAD

**Discussion by drakon660** [tye-v2/tye2#22] [General]
There is a need in my project to run legacy apps, a few, I want it to use a Tye2, I have released it locally and it works but there is one case regarding iisexpress which I need to use, basically it runs but I cannot kill a Tye2, the process doesn't handle ctrl+c signal. I was checking if I could fix it somehow, but couldn't make it, commandline lib is pretty old and new beta4 has also breaking changes, I went through issues and seams that MS don't care.  What is your opinion on that? I am think about creating some POC on Spectre.Console.

**Comment by razvangoga** [comment]
i'm not familiar with Specter.Console (i have heard about it), but i did try at one point to update the commandline lib, and it's not that simple.

I'm not opposed to a PoC but let's first update the app to net8 before we start making any code changes in it.

**Comment by drakon660** [comment]
We have this in another discussion.
